src/index.js

```diff
@@ -112,10 +112,9 @@ module.exports.ServerController = function (state) {
     accessors: {
       get: function (path) {
         path = typeof path === 'string' ? path.split('.') : path
-        while (path.length) {
-          state = state[path.shift()]
-        }
-        return state
+        return path.reduce(function (currentState, key) {
```

Comment:
Cannot read property 'setting' of undefined, when the path is 'head.setting'. In fact, simply no ‘head’.